integration/exec_test.go

```diff
@@ -361,12 +380,12 @@ func TestFreeze(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if state != configs.Paused {
-		t.Fatal("Unexpected state: ", state)
-	}
 	if err := container.Resume(); err != nil {
 		t.Fatal(err)
 	}
+	if state != configs.Paused {
```

Comment:
@mrunalp We get a state before calling resume, but check it after. I do this to not leave a container in a frozen state. Otherwise we will need to do cleanup by hand. For example here is a command from my bash_history file:
echo THAWED > /sys/fs/cgroup/freezer/system.slice/integration-test.scope/freezer.state